### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by jgiuliani**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/jgiuliani/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
\ No newline at end of file
```
Comment (jgiuliani) [inline ReadMe.md]: a comment!

Comment (jgiuliani) [inline ReadMe.md]: !fix

Comment (jgiuliani) [inline ReadMe.md]: !fixed

**Pull request merged**